incubator/spark-history-server/values.yaml

```diff
@@ -0,0 +1,54 @@
+replicaCount: 1
+
+namespace: spark-history-server
+createNamespace: true
+clusterRoleName: spark-hs
+clusterRoleBindingName: spark-hs
+
+image:
+  repository: lightbend/spark
+  tag: k8s-daily
+  pullPolicy: Always
```

Comment:
The tag should be a for a stable release and the `pullPolicy` should be `IfNotPresent`. This way the version used is repeatably used (e.g., in test and production) and caching is leveraged for images.

We have this in the review guidelines at https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#configuration

Reply:
Yes definitely! I was planning to use a Spark 2.4 image once it's released. But for now I can change the image and tag to use Spark 2.3. I'll update the `pullPolicy` too.